### ISSUE TITLE
The needs-sec-review label should block merges

### ISSUE THREAD

**Issue by fmarier** [brave/security-action#328]
When the security action flags a PR, we should prevent it from being merged until the label has been removed.

**Comment by thypon** [comment]
I'm not sure we want this for every repo. We can have this as a configurable option, wdyt?

**Comment by fmarier** [comment]
Good idea. Let's start with a few repos and so how that goes in practice.
